server/db/models/order.js

```diff
@@ -91,6 +91,12 @@ var OrderSchema = new mongoose.Schema({
     type: String,
     enum: Object.keys(shippingRates),
     default: "standard"
+  },
+
+  status: {
+    type: String,
+    enum: ["pending", "shipped", "cancelled", "returned"],
```

Comment:
What is "pending"? Is that before you've checked out and are possibly still shopping? We should probably add another status for when you've checked out (submitted?) but your order has not been shipped yet. 

Possibly: pending => submitted => shipped  then cancelled or returned